src/components/header/_Header.scss

```diff
@@ -0,0 +1,47 @@
+@import url("https://fonts.googleapis.com/css2?family=Alfa+Slab+One&display=swap");
+
+/*Variables*/
+$font-logo: "Alfa Slab One", cursive;
+$font-primry: "Manrope", sans-serif;
+$white: #ffffff;
+$black-one: #0000001F;
+$black-two: #00000029;
+
+/*Block*/
+.header {
+  width: 100%;
+  height: auto;
+}
+
+/*Element*/
+.header__logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 79px 0 23px 0;
```

Comment:
If we're going to make this UI responsive, please use `rem` instead of `px`

Reply:
Attended